src/SFML/Window/FreeBSD/JoystickImpl.cpp

```diff
@@ -300,7 +300,7 @@ Joystick::Identification JoystickImpl::getIdentification() const
 ////////////////////////////////////////////////////////////
 JoystickState JoystickImpl::JoystickImpl::update()
 {
-    while (read(m_file, m_buffer.data(), static_cast<size_t>(m_length)) == m_length)
+    while (read(m_file, m_buffer.data(), static_cast<std::size_t>(m_length)) == m_length)
```

Comment:
and this change on this line is to be reverted too.

Reply:
How come? This is C++ code. It calls a C function but it's weird to use exceptional rules here. The cognitive overhead to maintain these rules is not trivial.